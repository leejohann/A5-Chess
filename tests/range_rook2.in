move 7 6 7 4
move 0 1 0 2
move 7 7 7 5
quit
