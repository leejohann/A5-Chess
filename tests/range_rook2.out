RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
_ _ _ _ 
pppppppp
rnbqkbnr

it is White's turn
RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ p
_ _ _ _ 
ppppppp_
rnbqkbnr

it is Black's turn
RNBQKBNR
_PPPPPPP
P_ _ _ _
_ _ _ _ 
 _ _ _ p
_ _ _ _ 
ppppppp_
rnbqkbnr

it is White's turn
RNBQKBNR
_PPPPPPP
P_ _ _ _
_ _ _ _ 
 _ _ _ p
_ _ _ _r
ppppppp_
rnbqkbn 

it is Black's turn

